clar2wasm/tests/standard/unit_tests.rs

```diff
@@ -1713,3 +1713,17 @@ fn test_sqrti_int() {
         .call(&mut store, &[Val::I64(-1), Val::I64(-1)], &mut result)
         .expect_err("expected sqrti of negative integer");
 }
+
+#[test]
+fn bit_not_int() {
+    let (instance, mut store) = load_stdlib().unwrap();
+    let bitnot = instance.get_func(&mut store, "bit-not-int").unwrap();
+    let mut result = [Val::I64(0), Val::I64(0)];
+
+    // bit-not(0) = 0
```

Comment:
Wait... is that comment accurate? I don't get it

Reply:
fixed